include/boost/gil/image_processing/harris.hpp

```diff
@@ -0,0 +1,54 @@
+#ifndef BOOST_GIL_HARRIS_HPP
+#define BOOST_GIL_HARRIS_HPP
+
+#include <boost/gil/image_view.hpp>
+#include <boost/gil/typedefs.hpp>
+
+namespace boost { namespace gil {
+
+void find_corners_harris(
```

Comment:
Can you add inline comments (doxygen-style, preferably) to explain (in broad strokes) what the function is doing, and specifically, what the arguments represent ?

Reply:
I was thinking about writing a bigger document, as this requires quite a lot of math to understand.

Reply:
Both are needed: a design document that explains the algorithms, as well as inline documentation that explains the API. To be able to review the PR I need to have a basic understanding at least of what the arguments (and return values) are.

Reply:
Is it okay now?

Reply:
Thanks ! (You should build the docs (run `b2` inside the `doc/` subdirectory) to validate the formatting. I suspect the empty line between the comment block and the function definition prevents doxygen from parsing the comment as the function's documentation. But that's just from a casual look at the code.
The information itself is certainly useful. Thanks for adding it !